README.md

```diff
@@ -45,16 +45,41 @@ This repository uses `.deploy-output.sh` to generate the Editor's Draft on the `
 # Creating a new Working Draft
 
 To build a new WD and upload it to the W3C publishing system:
-- Make sure Bikeshed is installed locally
+- Register as a W3C member and join the Web Authentication working group. Note down your W3C USERNAME and PASSWORD to use in the command below. If you don't remember either one, please go [here](https://www.w3.org/accounts/recover) to retrieve it.
```

Comment:
I think this was accidentally pulled in during one of the historical rebases. Could you double-check all the changes on this branch and make sure they're related to the change you intend to make?

In particular, I think merge commits in the jeffh-fixup-algs-contd-3 branch are likely to confuse things.

Reply:
> double-check all the changes on this branch and make sure they're related to the change you intend to make?

done via the diff I link-to above.  looks fine to me.  I had to rebase onto master because master has been changing out from under this branch. 

> In particular, I think merge commits in the jeffh-fixup-algs-contd-3 branch are likely to confuse things.

am not sure what you mean by this.